src/drt/src/io/io.cpp

```diff
@@ -2074,11 +2074,14 @@ void io::Parser::readDb(odb::dbDatabase* db)
 
   if (VERBOSE > 0) {
     logger->report("");
-    frBox dieBox;
+    Rect dieBox;
     design->getTopBlock()->getDieBox(dieBox);
     logger->report("Design:                   {}",
                    design->getTopBlock()->getName());
-    logger->report("Die area:                 {}", dieBox);
+    // TODO Rect can't be logged directly
+    stringstream dieBoxSStream;
+    dieBoxSStream << dieBox;
+    logger->report("Die area:                 {}", dieBoxSStream.str());
```

Comment:
@maliberty I haven't found a better way to print a `Rect` to the logger because of the way `operator<<` only applies to dbStreams. Any suggestions?

Reply:
We should get rid of dbStreams.  In the interm you can add another overload for iostreams.

Reply:
I'll look into that in a different PR then.